.stylelintrc

```diff
@@ -0,0 +1,147 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      except: [
+        "blockless-after-same-name-blockless",
+        "first-nested",
```

Comment:
hmm, what is the value of this rule given that the only `@` styles that might go in a nested block would be `@extends` or `@includes`, which we won't be using anymore in our css? Also, I'm wondering why we only apply the exception to first-nested instead of all-nested, because this rule would imply that the following is fine:
```css 
{
    color: pink;
   
    @extends .foo;
    font-family: 'arial';
}
```
Presumably, if we are going to consider rules for nested `@` styles, we would probably also want a rule that says something like "if you're using `@extends` or `@includes`, it must come first in the nesting?